kola-denylist.yaml

```diff
@@ -21,6 +21,12 @@
     - centos-9
     - centos-10
 
+- pattern: ext.config.shared.security.lockdown
+  tracker: https://github.com/openshift/os/issues/1237
+  osversion:
+    - centos-9
+    - centos-10
```

Comment:
Minor: May be you can revert the changes related to this. Looks like the lines in which it was earlier has changed.

Reply:
@marmijo just asked me to move these lines to the top of file to group with the others that are related